devtools/conda-envs/test_min.yaml

```diff
@@ -1,17 +1,17 @@
-name: test_min
+name: test
```

Comment:
Renamed to `test` since we explicitly tell `conda-incubator/setup-miniconda@v2` to active an environment named `test`